.gitignore

```diff
@@ -21,3 +21,6 @@ metadata/
 **/dw.js
 **/dw.json
 .env
+
+# avoid publishing translation config
+crowdin.yml
```

Comment:
ok for me but, how do we share crowdin config now ?

Reply:
I keep this file in the dev repository too, so we just have to copy it on set up/ when needed.